hie-apply-refact/Haskell/Ide/ApplyRefactPlugin.hs

```diff
@@ -77,39 +78,20 @@ applyAllCmd = CmdSync $ \_ctxs req -> do
 applyHint :: FilePath -> Maybe Pos -> IO (Either String HieDiff)
 applyHint file mpos = do
   withTempFile $ \f -> do
-    -- absFile <- makeAbsolute file
-    -- hlint /tmp/Foo.hs --refactor --refactor-options="-o /tmp/Bar.hs --pos 2,8"
-
-    let
-      optsf = "-o " ++ f
-      opts = case mpos of
-        Nothing -> optsf
-        Just (r,c) -> optsf ++ " --pos " ++ show r ++ "," ++ show c
-    -- let hlintOpts = [file, "--quiet", "--refactor", "--refactor-options=" ++ opts ]
-    let hlintOpts = [file, "--quiet" ]
-    logm $ "applyHint=" ++ show hlintOpts
-    res <- catchException $ hlint hlintOpts
-    logm $ "applyHint:res=" ++ show res
-    -- res <- hlint hlintOpts
+    (flags,classify,hint) <- autoSettings
```

Comment:
You should rather use argSettings, so we can eventually pass through any extra flags the user has set.

See https://github.com/alanz/haskell-ide-engine/commit/97ccf826cb9f551b0e1b82ebbcc67a9c7189a644#diff-7b1a90d13e6fd0644f04b15c6c20befeR106, from my branch updating apply-refact